index.js

```diff
@@ -44,7 +44,7 @@ var cm = require("sdk/context-menu");
 
 cm.Item({
   label: "Send to mini player",
-  context: cm.SelectorContext('[href*="youtube.com"], [href*="youtu.be"]'),
+  context: cm.SelectorContext('[class*="yt-uix-sessionlink"], [href*="youtube.com"], [href*="youtu.be"]'),
```

Comment:
That makes sense. Thanks! 

New commit below. Should I squash them all into one?